tensorrt_llm/_torch/auto_deploy/models/custom/modeling_glm4_moe.py

```
@@ -0,0 +1,501 @@
+        self.norm_topk_prob = config.norm_topk_prob
+
+        self.weight = nn.Parameter(
+            torch.empty((self.n_routed_experts, config.hidden_size), dtype=torch.float32)
+        )
+        self.register_buffer(
+            "e_score_correction_bias",
+            torch.zeros(self.n_routed_experts, dtype=torch.float32),
+        )
+
+    def forward(self, hidden_states: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
+        bsz, seq_len, hidden_dim = hidden_states.shape
```

Ruff (0.15.10)

[warning] Unpacked variable `bsz` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)